src/diagram/components/quantity-node.scss

```
@@ -154,4 +154,16 @@ $gray-mid: #bcbcbc;
     padding: 2px 6px;
     border: 0 solid #999;
   }
+
+  .variable-description-area {
+    resize: none;
+    width: 100%;
+    text-align-last:left!important;
```

```suggestion
    text-align-last: left !important;
```
I think we can spare the additional spaces. 😉